src/test/java/NewPlaylistTestsUsingPOM.java

```diff
@@ -5,7 +5,7 @@
 import org.testng.annotations.Test;
 
 public class NewPlaylistTestsUsingPOM extends BaseTest {
-    private String notificationMessage = "Created playlist \"Iulia's Playlist1.\"";
+    private String notificationMessage = "Created playlist \"Iulia's Playlist.\"";
```

Comment:
This code is not used in the test and can be deleted `BasePage basePage = new BasePage(driver);`